app/js/services/situationService.js

```diff
@@ -187,8 +187,16 @@ angular.module('ddsApp').factory('SituationService', function($http, $sessionSto
 
         flattenPatrimoine: flattenPatrimoine,
 
+        getDemandeur: function(situation) {
+            return _.find(situation.individus, { role: 'demandeur' });
+        },
+
         hasEnfantScolarise: function(situation) {
             return !!_.find(situation.individus, { role: 'enfant', scolarite: 'college' }) || !!_.find(situation.individus, { role: 'enfant', scolarite: 'lycee' });
+        },
+
+        hasEnfant: function(situation) {
+            return !!_.find(situation.individus, { role: 'enfant' });
```

Comment:
Space after unary operators (one may consider double-bang construct as a unary operator).

In general, avoid that kind of idiomatic-but-unreadable casts. If you really do need to cast to a `Boolean`, use `Boolean(val)` rather than `!! val` (additionally, the higher cost of writing will make you think twice before doing it  ;)  ).